### PR TITLE
Fix messages queries with cluster cassandra

### PR DESCRIPTION
**Pull request by c-w**
Pull request relies on this schema change: https://github.com/CatalystCode/project-fortis-pipeline/pull/111

### CODE REVIEW

File: src/resolvers-cassandra/Messages/queries.js [hunk 1]
```diff
@@ -51,8 +93,8 @@ function byLocation(args, res) { // eslint-disable-line no-unused-vars
     AND externalsourceid = ?
     AND centroidlat = ?
     AND centroidlon = ?
-    AND eventtime >= ?
-    AND eventtime <= ?
+    AND eventime >= ?
```
Comment (Smarker) [inline src/resolvers-cassandra/Messages/queries.js]: We're not using `eventtime` anymore?

File: src/resolvers-cassandra/Messages/queries.js [hunk 2]
```diff
@@ -264,8 +242,16 @@ function event(args, res) { // eslint-disable-line no-unused-vars
...
   return new Promise((resolve, reject) => {
     translatorService.translate(args.sentence, args.fromLanguage, args.toLanguage)
-      .then(result => resolve({ translatedSentence: result.translatedSentence, originalSentence: args.sentence }))
-      .catch(reject);
+    .then(result => {
+      const translatedSentence = result.translatedSentence;
+      const originalSentence = args.sentence;
+
+      resolve({
+        translatedSentence,
```
Comment (Smarker) [inline src/resolvers-cassandra/Messages/queries.js:250]: You can have an object without properties?

Comment (c-w) [reply src/resolvers-cassandra/Messages/queries.js:250]: This is one of the ES6 niceties:

```js
const foo = 1;
{foo}
```

Is the same as:

```js
const foo = 1;
{foo: foo}
```

Only less verbose :)

File: src/resolvers-cassandra/Messages/queries.js (hunk 1 above)
Comment (c-w) [reply src/resolvers-cassandra/Messages/queries.js]: Good catch, I made this change to enable testing against the cluster but it should be reverted.

Comment (c-w) [reply src/resolvers-cassandra/Messages/queries.js]: Fixed in f736292.

File: src/resolvers-cassandra/Messages/queries.js (hunk 2 above)
Comment (Smarker) [reply src/resolvers-cassandra/Messages/queries.js:250]: Ok good to know!

**Review by Smarker — APPROVED** [review]
LGTM

**Pull request merged**